src/PyodideWorker.js

```diff
@@ -416,6 +476,14 @@ const PyodideWorker = () => {
 
     const lines = trace.split("\n");
 
+    // if the third from last line matches /File "__custom_open__\.py", line (\d+)/g then strip off the last three lines
+    if (
+      lines.length > 3 &&
```

Comment:
This feels a bit brittle but maybe I need more context, what are we trying to do here?

Reply:
Yeah is hideous, I admit it 😕 Basically, we're trying to make sure that any errors in opening files reference the user's file with the correct line number rather than the snippet that's providing the custom open function